src/SFA.DAS.FAT.Application.UnitTests/Courses/Queries/GetCourses/WhenGettingCourses.cs

```diff
@@ -39,20 +40,64 @@ public class WhenGettingCourses
         }
 
         [Test, MoqAutoData]
-        public async Task Then_Keywords_And_Sectors_Are_Passed_To_The_Service(
+        public async Task Then_The_Service_Is_Called_And_Keywords_Are_Returned(
```

Comment:
Test title might be slightly wrong?

`Then_The_Service_Is_Called_Passing_Keywords_From_Query`